CODEOWNERS

```diff
@@ -128,18 +128,18 @@ models/bert_large_performant @tt-aho @TT-BrianLiu
 models/metal_BERT_large_11 @tt-aho @TT-BrianLiu
 functional_*/ @eyonland @arakhmati @cfjchu @xanderchin
 models/demos @eyonland @arakhmati @cfjchu @xanderchin
-models/demos/wormhole @uaydonat
-models/demos/t3000 @uaydonat
-models/demos/falcon7b @skhorasganiTT
-models/demos/mamba @esmalTT
-models/demos/wormhole/falcon7b @skhorasganiTT
-models/demos/wormhole/mistral7b @yieldthought
-models/demos/t3000/falcon40b @johanna-rock-tt
-models/demos/t3000/falcon7b @skhorasganiTT
-models/demos/t3000/llama2_70b @cglagovichTT
-models/demos/t3000/llama3_70b @cglagovichTT
-models/demos/t3000/mixtral8x7b @yieldthought
-models/demos/grayskull @boris-drazic
+models/demos/wormhole @uaydonat @eyonland
```

Comment:
Can @nsmithtt and I be added to the wormhole demos as well?

Reply:
Just `models/demos/wormhole`?
Or  `models/demos/wormhole`, `models/demos/wormhole/falcon7b`, and `models/demos/wormhole/mistral7b`?

Reply:
I need `stable_diffusion`, which will be moved into `models/demos/wormhole`

Reply:
Ok. Done.